data/strings/types_strings.txt

```diff
@@ -18559,6 +18586,7 @@
     fr = Chemin de fer abandonné
     ja = 廃線
     nl = Voormalig treinspoor
+    pl = Rozebrane tory kolejowe
```

Comment:
```suggestion
    pl = Pozostałości po rozebranych torach kolejowych
```

Reply:
completely gone ones are not mappable in OSM

Reply:
I was basing my translation on OSM Wiki description and photos
> A tag to map former railways, where the rails have been removed but the route is still visible with ties/sleepers, signals or other railroad right of way signs or infrastructure. 
So not without traces, actually.

Reply:
This category has been since deleted.